megazords/ios/DEPENDENCIES.md

```diff
@@ -889,6 +910,38 @@ OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR
 IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 DEALINGS IN THE SOFTWARE.
 
+```
+-------------
+## MIT License: cargo_metadata
```

Comment:
With sufficient effort, we might be able to relegate these to dev dependencies in a way that our dependency-script can ignore them. I haven't looked into it.